tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1107,27 +1107,7 @@ def adjust_hue(image, delta, name=None):
     # Remember original dtype to so we can convert back if needed
     orig_dtype = image.dtype
     flt_image = convert_image_dtype(image, dtypes.float32)
-
-    # TODO(zhengxq): we will switch to the fused version after we add a GPU
-    # kernel for that.
-    fused = os.environ.get('TF_ADJUST_HUE_FUSED', '')
```

Comment:
For safety concern, leave the default as "fused". If the user specified "0", or "false", use the old code path. 

After a few weeks, if we don't see anything broken, we will remove the old path.